data/json/mutations/mutations.json

```diff
@@ -4672,7 +4674,7 @@
     "points": -1,
     "visibility": 10,
     "ugliness": -2,
-    "description": "You have a very large and very beautiful pair of brightly-colored wings.  They can't lift you, and they make balancing tricky, but they certainly catch air and attention!",
+    "description": "You have a very large, very beautiful pair of brightly-colored wings.  They can't lift you, and they make balancing tricky, but they certainly catch air and attention!",
```

Comment:
```suggestion
    "description": "You have a very large and beautiful pair of brightly-colored wings.  They can't lift you, and they make balancing tricky, but they certainly catch air and attention!",
```

Reply:
What's the reason for the reversion? Splitting the clause here with a comma reads more naturally -- to me, at least.

Reply:
Two "very"s in a row? Perhaps if an alternative superlative was found.

Reply:
Repetition isn't always a sin. Linguistically, it can reinforce what's being repeated without bolting something else on. "Very large and beautiful" avoids the repetition, but it feels...stilted, I guess is the word.

Reply:
```suggestion
    "description": "You have a very large pair of brightly-colored, very beautiful wings.  They can't lift you, and they make balancing tricky, but they certainly catch air and attention!",
```

Reply:
I like it!